rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -1057,6 +1058,18 @@ public long getLockingCap(Object[] args) {
         return lockingCap.getValue();
     }
 
+    public byte[] getActivePowpegRedeemScript(Object[] args) {
+        logger.trace("getActivePowpegRedeemScript");
+
+        Script activePowpegRedeemScript = bridgeSupport.getActivePowpegRedeemScript();
+
+        if (activePowpegRedeemScript == null){
+            return new byte[]{};
+        }
```

Comment:
We can remove this check now since we are returning a Script

Reply:
Good catch